externs/ngeox.js

```diff
@@ -49,3 +49,33 @@ ngeox.interaction.MeasureOptions.prototype.style;
  * @type {ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined}
  */
 ngeox.interaction.MeasureOptions.prototype.sketchStyle;
+
+
+/**
+ * Namespace.
+ * @type {Object}
+ */
+ngeox.profile;
+
+
+/**
+ * Options for the profile.
+ *  @typedef {{
+ *    extractor: ngeox.profile.ProfileExtractor,
+ *    light: (boolean|undefined),
+ *    hoverCallback: (function(Object)|undefined),
+ *    outCallback: (function()|undefined),
+ *    zKeys: (Array.<string>|undefined)
+ *  }}
+ */
+ngeox.profile.ProfileOptions;
```

Comment:
I thought it was only a requirement of the documentation generation tool used by Ol3.
Where is defined `ngeox.interaction.MeasureOptions.prototype`?